src/main/java/com/puppycrawl/tools/checkstyle/Main.java

```diff
@@ -724,8 +724,8 @@ private static final class CliOptions {
         /** Option that controls whether to print the AST of the file. */
         @Option(names = {"-t", "--tree"},
                 description = "Prints Abstract Syntax Tree(AST) of the checked file. The option "
```

Comment:
`This option is used to display the Abstract Syntax Tree (AST) of the specified file. It can only be used on a single file and cannot be combined with other options.`

Reply:
Do you want the options `-T , -j , -J` also to be updated in a similar way in this PR ?

Reply:
Do they need to be improved?

Reply:
```
-t, --tree - Prints Abstract Syntax Tree(AST) of the checked file. The option cannot be used other options and requires exactly one file to run on to be specified.
-T, --treeWithComments - Prints Abstract Syntax Tree(AST) with comment nodes of the checked file. The option cannot be used with other options and requires exactly one file to run on to be specified.
-J, --treeWithJavadoc - Prints Abstract Syntax Tree(AST) with Javadoc nodes and comment nodes of the checked file. Attention that line number and columns will not be the same as it is a file due to the fact that each javadoc comment is parsed separately from java file. The option cannot be used with other options and requires exactly one file to run on to be specified.
-j, --javadocTree - Prints Parse Tree of the Javadoc comment. The file have to contain only Javadoc comment content without including '/**' and '*/' at the beginning and at the end respectively. The option cannot be used other options and requires exactly one file to run on to be specified.
```
The wording for all 4 options is similarly used more or less. Updating all 4 to follow your new suggestion for `-t` would make it uniform and clear, I believe

Reply:
I just ran what was there through ChatGPT and said "Improve this..." :) You can do the with the other descriptions here if you would like. I do notice some grammatical errors in them, so they could certainly be improved, if we only did that.